### PR TITLE
SEO-179381-MAUI-Text-Input-Layout-External-Linking

### PR DESCRIPTION
**Pull request by berylouma**
|Title|Description|
|--|--|
|Task Category| .NET MAUI TextInputLayout|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|External linking PR link| Ext -https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-text-input-layout/pull/2|
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/179663/|
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Text-Input-Layout--Money-Keywords.aspx|

Worked on 17 keywords in the .NET MAUI Text Input Layout and am keeping track of all keyword details in the link below:-
[https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/Keyword-Plan---.NET-MAUI-Text-Input-Layout.aspx]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,57 @@
...
-
-Refer to the following documentation for the Syncfusion .NET MAUI Text Input Layout control: 
-https://help.syncfusion.com/maui/textinputlayout/getting-started
-
-Check out this example of the Syncfusion .NET MAUI Text Input control on GitHub: 
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/TextInputLayout
+The [.NET MAUI Text Input Layout](https://www.syncfusion.com/maui-controls/maui-textinputlayout?utm_source=github&utm_medium=listing&utm_campaign=maui-text-input-layout-github-samples) or Label Entry is a container control that allows users to add floating labels, password toggle icons to show or hide passwords, leading and trailing icons, and assistive labels such as error messages and help text on top of input controls.A quick-start project that helps you to integrate a Text input layout control in the .NET MAUI application. This project contains simple code customizations like how to add floating labels, password toggle icons to show or hide passwords, and assistive labels like error messages and help text on top of input controls.
```
Comment (Sharly-Atieno) [inline README.md]: Give a space here..
image

File: README.md [hunk 2]
```diff
@@ -1,14 +1,57 @@
...
 
 ## Project prerequisites
 Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
 
 ## How to run this application
-To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-text-input-layout` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
\ No newline at end of file
+To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-text-input-layout` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+
```
Comment (Sharly-Atieno) [inline README.md]: Remove extra space here..

Comment (berylouma) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (berylouma) [reply README.md]: Done

**Review by Sharly-Atieno — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**